EmailNotification/__tests__/utils.test.ts

```diff
@@ -78,18 +79,48 @@ describe("removeLinks", () => {
   test("should return the string without the url if a simple url is contained", () => {
     const simpleLink = "https://asimplelink.com/";
     const baseText = `A simple text`;
-    expect(removeLinks(`${baseText}${simpleLink}`)).toBe(baseText);
+    expect(removeLinks(`${baseText} ${simpleLink}`)).toBe(
+      `${baseText} [link rimosso]`
+    );
+  });
+
+  test("should return the string without the url if more than one simple url are contained", () => {
+    const simpleLink = "https://asimplelink.com/";
+    const baseText = `A simple text`;
+    expect(
+      removeLinks(
+        `${baseText} ${simpleLink} this is another link ${simpleLink}`
+      )
+    ).toBe(`${baseText} [link rimosso] this is another link [link rimosso]`);
   });
 
   test("should return the string without the url if an url with query params is contained", () => {
     const simpleLink = "https://asimplelink.com/?qp1=value";
     const baseText = `A simple text`;
-    expect(removeLinks(`${baseText}${simpleLink}`)).toBe(baseText);
+    expect(removeLinks(`${baseText} ${simpleLink}`)).toBe(
+      `${baseText} [link rimosso]`
+    );
   });
 
   test("should return the string without the url if an url with query params and # is contained ", () => {
     const simpleLink = "https://asimplelink.com/?qp1=value#header";
     const baseText = `A simple text`;
-    expect(removeLinks(`${baseText}${simpleLink}`)).toBe(baseText);
+    expect(removeLinks(`${baseText} ${simpleLink}`)).toBe(
+      `${baseText} [link rimosso]`
+    );
+  });
+});
+
+describe("invalidateClickableLinks", () => {
+  test("should return the same string if no period are provided", () => {
+    expect(invalidateClickableLinks("a simple text with no perios")).toBe(
+      "a simple text with no perios"
+    );
+  });
+
+  test("should add a zero-width space before every '.' character", () => {
```

Comment:
you say before, but it seems that it's after.

Reply:
[22dbf96](https://github.com/pagopa/io-functions-services/pull/297/commits/22dbf96d45f66a64fd34a79160e0651cc02f0117)